templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -212,6 +212,13 @@
 					{{end}}
 				</a>
 				<div class="menu" data-action="update" data-issue-id="{{$.Issue.ID}}" data-update-url="{{$.RepoLink}}/issues/projects">
+					<div class="header" style="text-transform: none;font-size:16px;">{{.locale.Tr "repo.issues.new.add_project_title"}}</div>
```

Comment:
Please avoid using style. There should be a class you can use to do this

Reply:
I copied the code exactly from `new_form.tmp`, and within this file the same `style` attribute is used for reviewers, labels, milestones.

They can all be changed together, but should that be done in the same PR as this bugfix?

Reply:
Agree, this PR is not guilty if it's consistent 😉.

Reply:
I think we can remove this div, looks like it's unnecessary.

Reply:
Or we can move `clear projects` on top of `set projects` with a divider.

Reply:
To be clear, do you want me to make any further changes in this PR? Or can I just leave this as is since it matches the same menu in `new_form.tmpl` exactly, and so any further changes make more sense as a separate PR?